### PR TITLE
[Templating] [Template] Render a block with the `#[Template]` attribute

### LINKED ISSUE #20123: [TwigBridge] Render a `block` via the `#[Template]` attribute
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#58028
| PR author(s) | @smnandre
| Merged in    | 7.2

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.2 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by smnandre**
Document  [symfony/symfony#58028](https://github.com/symfony/symfony/pull/58028) 
and closes #20123

### CODE REVIEW

File: templates.rst [hunk 1]
```diff
@@ -632,6 +632,27 @@ This might come handy when dealing with blocks in
...
+    {
+        #[Template('product.html.twig', block: 'price_block')]
+        public function price(): array
+        {
+            return [
+                // ...
+            ];
+        }
+    }
+
```
Comment (OskarStark) [inline templates.rst:655]: Please add a versionadded 7.2 directive, thanks

Comment (smnandre) [reply templates.rst:655]: Added, thx!

File: templates.rst [hunk 2]
```diff
@@ -632,6 +632,31 @@ This might come handy when dealing with blocks in
...
+.. versionadded:: 7.2
+
+    The ``#[Template]`` attribute's ``block``argument was introduced in Symfony 7.2.
```
Comment (xabbuh) [inline templates.rst]:
```suggestion
    The ``#[Template]`` attribute's ``block`` argument was introduced in Symfony 7.2.
```

Comment (smnandre) [reply templates.rst]: Thank you!

**Review by OskarStark — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Simon!

**Pull request merged**